fe/src/main/java/org/apache/doris/load/DeleteHandler.java

```diff
@@ -450,7 +451,22 @@ public DeleteJob getDeleteJob(long transactionId) {
         return idToDeleteJob.get(transactionId);
     }
 
-    private void checkDeleteV2(OlapTable table, Partition partition, List<Predicate> conditions, List<String> deleteConditions, boolean preCheck)
+    private SlotRef getSlotRef(Predicate condition) {
+        SlotRef slotRef = null;
+        if (condition instanceof BinaryPredicate) {
+            BinaryPredicate binaryPredicate = (BinaryPredicate) condition;
+            slotRef = (SlotRef) binaryPredicate.getChild(0);
```

Comment:
can slotRef be the second child of binaryPreidicate?

Reply:
not allowed